content/work/sass-core.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Sass Core Language
+sub: Implementing new language features
+date: 2023-05-22
+end: 2024-02-20
+client: &client Google
+logo: google
+image:
+  svg: logos/sass
+list_tag: Sass
+people:
+  - &natalie
+    name: Natalie Weizenbaum
+    title: Lead designer and developer of Sass
+    face: natalie-weizenbaum.webp
+    url: https://github.com/nex3
+    venue: *client
+press:
+  - text: |
+      OddBird's team helped power out
+      several new Sass features
+      that would have taken the core team alone
+      years longer.
+      **They dropped in like a task force**,
+      learned the ropes of programming language
+      design and development,
+      and made substantial improvements to the product.
+      We'd love to work with them again!
+    <<: *natalie
```

Comment:
Do we want to show this quote on this page?

Reply:
Added in https://github.com/oddbird/oddleventy/pull/540/commits/259a7e0ec3a14b8a844a8bcf441e65929334c96e